CRM/Case/BAO/Case.php

```diff
@@ -15,6 +15,8 @@
  * @copyright CiviCRM LLC https://civicrm.org/licensing
  */
 
+use Civi\Api4\Activity;
+
```

Comment:
@mattwire looks like the style checker is being picky about whitespace and wants you to remove this extra blank line.